app/src/main/java/com/example/ecarchargeinfo/map/domain/repository/allmarker/AllMarkerRepositoryImpl.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.ecarchargeinfo.map.domain.repository.allmarker
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.drawable.BitmapDrawable
+import androidx.core.content.res.ResourcesCompat
+import com.example.ecarchargeinfo.R
+import com.example.ecarchargeinfo.map.domain.model.MapConstants
+import com.example.ecarchargeinfo.map.domain.util.MyItem
+import com.example.ecarchargeinfo.retrofit.model.charger.ChargerInfo
+import com.google.android.gms.maps.model.BitmapDescriptorFactory
+import com.google.android.gms.maps.model.LatLng
+import dagger.hilt.android.qualifiers.ApplicationContext
+import javax.inject.Inject
+
+class AllMarkerRepositoryImpl @Inject constructor(@ApplicationContext private val context: Context) :
+    AllMarkerRepository {
+    override fun getAllMarker(list: List<ChargerInfo>): ArrayList<MyItem> {
+        val markerArray = ArrayList<MyItem>()
+        list.forEachIndexed { index, chargerInfo ->
+            if (index > 0) {
+                if (list[index - 1].csNm == chargerInfo.csNm) return@forEachIndexed
+            }
+            val markerImage = getMarkerImage(chargerInfo.chargeTp)
+            val resizeImage = getResizedImage(markerImage)
+
+            val item = MyItem(
+                LatLng(chargerInfo.lat.toDouble(), chargerInfo.longi.toDouble()),
+                chargerInfo.csNm,
+                chargerInfo.cpStat,
+                BitmapDescriptorFactory.fromBitmap(resizeImage),
```

Comment:
```suggestion
                BitmapDescriptorFactory.fromBitmap(
                  getResizedImage(
                    getMarkerImage(
                      chargeTp = chargerInfo.chargeTp
                    )
                  )
                )
```

굳이 변수로 한번 받을 필요 없이 이런식으로 바꿔주시면 더 좋을 것 같습니다.